great_expectations/execution_engine/sqlalchemy_dialect.py

```diff
@@ -37,7 +37,7 @@ def __hash__(self: GESqlDialect):
         return hash(self.value)
 
     @classmethod
-    def _missing_(cls, value: Any) -> None:
+    def _missing_(cls, value: Any) -> GESqlDialect:
```

Comment:
Doesn't returning something from super imply that a non-GESqlDialect Enum type could be returned?

Reply:
Yes, `Any` is probably more appropriate here.